concourse/model/traits/component_descriptor.py

```diff
@@ -27,39 +27,24 @@
 from concourse.model.base import (
     AttribSpecMixin,
     AttributeSpec,
+    EnumWithDocumentation,
     ModelValidationError,
     ScriptType,
     Trait,
     TraitTransformer,
 )
 
 
-class ValidationPolicy(AttribSpecMixin, enum.Enum):
-    NOT_EMPTY = "not_empty"
-    FORBID_EXTRA_ATTRIBUTES = "forbid_extra_attributes"
+class ValidationPolicy(EnumWithDocumentation):
+    NOT_EMPTY = (
+        'not_empty',
```

Comment:
I did not try: is it possible to pass those as named parameters?

Reply:
Unfortunately not. As a workaround, NamedTuples should provide the added explicitness. I'll experiment with them.

Reply:
ok - as discussed earlier: it is OK with me to use two-tuples;